### PR TITLE
Add PR Template

### LINKED ISSUE #4: Add Pull Request Template

### PR DESCRIPTION
**Pull request by est271**
Fixes #4 : Add Pull Request Template

I tested the template on my repo and it works. It doesn't come up on this PR because it hasn't been merged but it should look like what is shown below, feel free to modify the .md file as you wish:

- [x] Fixes issue #IssueNo : "Name of issue here"
- [x] other checks

## Description:
"Insert description of Pull Request here"

### CODE REVIEW

**Comment by deepak2431** [comment]
@est271 Hi, Thanks a lot for your pull request. I think this template can be made more informative. Can you some of the open source organization PR template. As for example to add screenshot of the issue fixed can be added.
Thanks!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+- [ ] Fixes issue #IssueNo : "Name of issue here"
+- [x] other checks
```
Comment (deepak2431) [inline .github/PULL_REQUEST_TEMPLATE.md]: o should be in capital letter.

**Comment by est271** [comment]
@deepak2431,
No worries, I have added the requested changes and some more informative descriptions that are similar to what I saw on other open source projects. Next time be more specific about requested changes when you open an issue. Below is a screenshot of the current template

PR

**Comment by deepak2431** [comment]
@est271 Looks good to me. Thanks!

**Pull request merged**